.mailmap

```diff
@@ -237,6 +239,7 @@ Matheus Marchini <matheusdot@gmail.com> <matheus@sthima.com.br>
 Matheus Marchini <matheusdot@gmail.com> <matheus@sthima.com>
 Matt Lang <matt@mediasuite.co.nz> matt-in-a-hat <matt@mediasuite.co.nz>
 Matt Reed <matthewreed26@gmail.com> matthewreed26 <matthewreed26@gmail.com>
+Matteo Collina <matteo.collina@gmail.com> <hello@matteocollina.com>
```

Comment:
@zkat Can you indicate which email is your preferred one? The gmail one got into the mix because of a co-authored commit.